code/game/objects/items/weapons/melee/energy_melee_weapons.dm

```diff
@@ -251,13 +242,59 @@
 		return -1
 	return TRUE
 
+//////////////////////////////
+// MARK: SAW
+//////////////////////////////
+/// Used by medical Syndicate cyborgs
+/obj/item/melee/energy/sword/cyborg/saw
+	name = "energy saw"
+	desc = "For heavy duty cutting. It has a carbon-fiber blade in addition to a toggleable hard-light edge to dramatically increase sharpness."
+	force_on = 30
+	force = 18 //About as much as a spear
+	sharp = TRUE
+	hitsound = 'sound/weapons/circsawhit.ogg'
+	icon = 'icons/obj/surgery.dmi'
+	icon_state = "esaw_0"
+	icon_state_on = "esaw_1"
+	hitcost = 75 //Costs more than a standard cyborg esword
+	item_color = null
+	w_class = WEIGHT_CLASS_NORMAL
+	light_color = LIGHT_COLOR_WHITE
+	tool_behaviour = TOOL_SAW
+
+/obj/item/melee/energy/sword/cyborg
+	var/hitcost = 50
```

Comment:
Could you move this above the type definition of the saw?

Reply:
Done.